doc/obc-provisioner.md

```diff
@@ -101,7 +101,39 @@ spec:
   generateBucketName: my-bucket
   storageClassName: noobaa.noobaa.io
   additionalConfig:
-    replication-policy: [{ "rule_id": "rule-2", "destination_bucket": "first.bucket", "filter": {"prefix": "bc"}}]
+    replicationPolicy: [{ "rule_id": "rule-2", "destination_bucket": "first.bucket", "filter": {"prefix": "bc"}}]
+```
+
+# OBC with an NSFS account config
+
+It is possible to create an OBC on top of NSFS. In order to do this, the user has to provide relevant data in `nsfsAccountConfig` under the `spec.additionalConfig` key.
+
+The config needs to contain the following fields so that NooBaa can access the filesystem:
+gid - the group ID of the account that should be mimicked within the filesystem
+uid - the user ID of the account that should be mimicked within the filesystem
+OR
+distinguished_name - the distinguished name of the account that should be mimicked within the filesystem
+
+These optional parameters can be provided by the user to further configure the NSFS account:
+new_buckets_path - the filesystem path that should be 'mounted' as the bucket's root (`path` parameter in the CLI)
```

Comment:
@Neon-White I don't think that we would like to let the user create other bucket's in the OBC's path..
IMO we should not let this account create any buckets since it's created for accessing this OBC. WDYT?

Reply:
Actually the current behavior of this account is [AllowBucketCreation=false](https://github.com/noobaa/noobaa-operator/pull/1342/files#diff-fc66f732e7eb85134eb34fcca0e5f6a5de879ec044b8e0d194de1c31e35b3c34R589) so this means that new_buckets_path has no meaning, thanks for removing it